Makefile

```
@@ -1,52 +1,66 @@
-# Rule to compile the kernel
-$(K_OBJ): $(K_SRC) 
-	$(CC) $(CFLAGS) -c $(K_SRC) -o $(K_OBJ)
-
-# Rule to build the bootloader
-boot/boot.bin: boot/boot.asm
-	$(AS) -f bin boot/boot.asm -o boot/boot.bin
-
-# Rule to clean up build files
-clean:
-	rm -f boot/*.bin kernel/*.o $(KERNEL_ELF) $(KERNEL_BIN) $(OS_IMG)
+CC=gcc
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)